pkg/daemon/update.go

```diff
@@ -65,6 +65,9 @@ const (
 	// ImageRegistryDrainOverrideConfigmap is the name of the Configmap a user can apply to force all
 	// image registry changes to not drain
 	ImageRegistryDrainOverrideConfigmap = "image-registry-override-drain"
+
+	// pinnedImagePathRegex is the regex MCO expects will contain pinned image updates.
+	pinnedImagePathRegex = `^\/etc\/crio\/crio\.conf\.d\/[0-9]{2}-pinned-images$`
```

Comment:
PinnedImageSet controller seems to be writing ignition config to `CRIODropInFilePathPinnedImages = "/etc/crio/crio.conf.d/01-pinned-images"` path. Is this an hardcoded path and something we want to fix here or this still makes sense?

Reply:
I think its hardcoded honestly we only want to reason about a single set on disk IMO. As we are enforcing a single set per pool I think the path can be static and we can adjust down the road if it becomes nessisary.